api/interest/documentation/1.0.0/interest.json

```diff
@@ -648,7 +648,8 @@
                 "meetingLinkURLs",
                 "meetingTimes",
                 "keywords",
-                "team"
+                "team",
+                "isListed"
```

Comment:
Can you elaborate how `isListed` is going to be used?

Reply:
@chungthuang it's basically going to enable project leaders to hide the project from the projects aggregator (/projects page) but still accessible through the link (/projects/slug of project)